docker/rasterize-chrome/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM demisto/python3-deb:3.7.4.1163
+
+COPY requirements.txt .
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python-dev \
+  wget \
+&& pip install --no-cache-dir -r requirements.txt \
+&& apt-get purge -y --auto-remove \
+  gcc \
+  python-dev \
+&& rm -rf /var/lib/apt/lists/*
+
+# Install Chrome
+RUN apt-get update
+RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+RUN dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install
```

Comment:
Combine the above three commands into one and add cleanup. Also, not sure about the apt-get install part and if it is needed.